src/Appwrite/Utopia/Response/Model/Rule.php

```diff
@@ -84,11 +84,16 @@ public function __construct()
                 'example' => 'main',
             ])
             ->addRule('status', [
-                'type' => self::TYPE_ENUM,
-                'description' => 'Domain verification status. Possible values are "created", "verifying", "verified" and "unverified"',
-                'default' => 'created',
-                'example' => 'verified',
-                'enum' => ['created', 'verifying', 'verified', 'unverified'],
+                'type' => self::TYPE_STRING,
+                'description' => 'Domain verification status. Possible values are "' . RULE_STATUS_VERIFICATION_FAILED . '", "' . RULE_STATUS_GENERATING_CERTIFICATE . '", "' . RULE_STATUS_SUCCESSFUL . '" and "' . RULE_STATUS_GENERATION_FAILED . '"',
+                'default' => '',
+                'example' => RULE_STATUS_SUCCESSFUL,
+            ])
+            ->addRule('verificationLogs', [
```

Comment:
Can this be just simply `logs`?

Reply:
We use logs for certificate logs. but yes, technically you dont ever see certificate logs before verification finishes. so we could re-use same attribute 👍